src/components/SearchResultsItem/SearchResultsItem.tsx

```diff
@@ -1,5 +1,4 @@
 import { FC } from "react";
-// import * as cheerio from "cheerio";
```

Comment:
Is this being used anywhere else in the code base? Should remove the library if not

Reply:
removed